2_observations/out/realsat_monthly_surface_area.csv

```diff
@@ -0,0 +1,553 @@
+reservoir,date,area_m2
+Cannonsville,1984-06-15,15957900
```

Comment:
hmmm I might have messed this up, b/c I remove the file from the .gitignore yesterday, before I deleted it 😟

I deleted it now and committed the deletion, but I can't commit any changes to the .gitignore b/c `!2_observations/out/realsat_monthly_surface_area.csv` had already been removed....

Reply:
Should I add `!2_observations/out/realsat_monthly_surface_area.csv` back to the .gitignore, commit the file, delete it, commit the deletion, remove it from .gitignore, and commit the changes to .gitignore?

Reply:
Yes, that's the way I've been able to solve such issues in the past. It feels painful to go back and forth (though in reality it shouldn't take too long). On the bright side, the pain will likely help you remember how to avoid the issue more often in the future =)

Reply:
Ok! I think we're good to go now